pkg/inspector/image-inspector.go

```diff
@@ -12,14 +12,18 @@ import (
 	"net/http"
 	"os"
 	"path"
+	"path/filepath"
 	"strings"
 	"time"
 
 	"archive/tar"
 	"crypto/rand"
 
+	"github.com/containerd/containerd/mount"
 	docker "github.com/fsouza/go-dockerclient"
 	"github.com/openshift/image-inspector/pkg/openscap"
+	"golang.org/x/net/context"
```

Comment:
same thing about `context` here as in above comment @legionus. just wondering if there's a reason we're preferring golang.org/x's version of context. i'm not sure if there are any differences

Reply:
@ilackarms There are no differences. I'll use the context from stdlib.